api/src/multiaccessy/accessy.py

```diff
@@ -541,14 +542,23 @@ def fill_membership_id(user: AccessyMember) -> None:
             user.membership_id = data["id"]
 
         threads = []
+        user_ids = list(user_ids)
+        thread_count = min(4, len(user_ids))
```

Comment:
I don't follow this completely. Doesn't this still create one thread per user_id as before?

Reply:
No, it creates at most 4 threads.